sortedm2m/templates/sortedm2m/sorted_checkbox_select_multiple_widget.html

```diff
@@ -1,12 +1,12 @@
 {% load i18n static %}
+
 <div class="sortedm2m-container">
 
     <p class="selector-filter">
         <img src="{% static "sortedm2m/selector-search.gif" %}" alt="" title="{% trans "Type into this box to filter down the list." %}" />
         <input type="text" placeholder="{% trans "Filter" %}" />
     </p>
-
-    <ul>
+    <ul class="hide">
```

Comment:
Could we make the `hide` class be added by JS on load? The problem here I see is if you don't have JS enabled you will never see the checkboxes ... I'm not sure if I'm living in the 1990s in that regard, what do you think?